src/code/views/file-dialog-tab-view.ts

```diff
@@ -93,16 +96,15 @@ const FileList = createReactClassFactory({
     if (isSubFolder) {
       list.push((div({key: 'parent', className: 'selectable', onClick: this.parentSelected}, (italic({className: 'icon-paletteArrow-collapse'})), this.props.folder.name)))
     }
-    for (i = 0; i < this.props.list.length; i++) {
+    for (let i = 0; i < this.props.list.length; i++) {
```

Comment:
As far as I can tell, this was previously modifying the `i` imported from `ReactDOMFactories` above. :scream:

Reply:
You have heard of iterative italics right? :)